Index: adaptor.h
===================================================================
RCS file: /var/cvsroot/burn/adaptor.h,v
retrieving revision 1.4
diff -u -r1.4 adaptor.h
--- adaptor.h	16 Sep 2006 18:33:36 -0000	1.4
+++ adaptor.h	8 Apr 2008 18:40:42 -0000
@@ -8,6 +8,7 @@
 #ifndef VDR_BURN_ADAPTOR_H
 #define VDR_BURN_ADAPTOR_H
 
+#include <algorithm>
 #include <iterator>
 #include <vdr/epg.h>
 #include <vdr/tools.h>
Index: filter.h
===================================================================
RCS file: /var/cvsroot/burn/filter.h,v
retrieving revision 1.5
diff -u -r1.5 filter.h
--- filter.h	16 Sep 2006 18:33:36 -0000	1.5
+++ filter.h	8 Apr 2008 18:40:42 -0000
@@ -9,6 +9,7 @@
 #define VDR_BURN_FILTER_H
 
 #include "common.h"
+#include <algorithm>
 #include <iterator>
 
 namespace vdr_burn
Index: iconvwrapper.c
===================================================================
RCS file: /var/cvsroot/burn/iconvwrapper.c,v
retrieving revision 1.3
diff -u -r1.3 iconvwrapper.c
--- iconvwrapper.c	16 Sep 2006 18:33:36 -0000	1.3
+++ iconvwrapper.c	8 Apr 2008 18:40:42 -0000
@@ -7,6 +7,8 @@
 
 #include "iconvwrapper.h"
 #include <cerrno>
+#include <cstdlib>
+#include <cstring>
 #include <sstream>
 
 namespace iconvwrapper
Index: menuitems.h
===================================================================
RCS file: /var/cvsroot/burn/menuitems.h,v
retrieving revision 1.15
diff -u -r1.15 menuitems.h
--- menuitems.h	19 Sep 2006 18:51:35 -0000	1.15
+++ menuitems.h	8 Apr 2008 18:40:42 -0000
@@ -14,6 +14,7 @@
 #include "tracks.h"
 #include <algorithm>
 #include <iterator>
+#include <limits>
 #include <numeric>
 #include <memory>
 #include <string>
Index: proctools/logger.cc
===================================================================
RCS file: /var/cvsroot/burn/proctools/logger.cc,v
retrieving revision 1.3
diff -u -r1.3 logger.cc
--- proctools/logger.cc	16 Sep 2006 18:33:38 -0000	1.3
+++ proctools/logger.cc	8 Apr 2008 18:40:42 -0000
@@ -6,6 +6,7 @@
 #include <sstream>
 #include <algorithm>
 #include <cerrno>
+#include <cstring>
 
 namespace proctools
 {
