--- burn/jobs.c.orig	2015-03-17 17:57:47.326448269 +0100
+++ burn/jobs.c	2015-03-17 18:14:18.232493156 +0100
@@ -263,7 +263,11 @@
 	std::string recording::get_graft_point() const
 	{
 		return proctools::format("{0}={1}")
+#if VDRVERSNUM >= 20102
+			   % m_fileName.substr(std::string(cVideoDirectory::Name()).length())
+#else
 			   % m_fileName.substr(std::string(VideoDirectory).length())
+#endif
 			   % m_fileName;
 	}
 
