--- vdr-burn-0.3.1/burn/config/vdrburn-dvd.conf.orig	2025-11-21 08:48:12.959074579 +0100
+++ vdr-burn-0.3.1/burn/config/vdrburn-dvd.conf	2025-11-21 08:48:43.775287771 +0100
@@ -1,5 +1,5 @@
 # settings for vdrburn-dvd.sh
 
 # Java and Project X
-#export JAVA_EXE=/usr/bin/java
-export PROJECTX_JAR=/usr/share/java/ProjectX.jar
+export JAVA_EXE=/usr/bin/java
+export PROJECTX_JAR=/usr/lib/java/ProjectX.jar
--- vdr-burn-0.3.1/burn/chain-archive.c.orig	2025-11-21 08:49:13.893355311 +0100
+++ vdr-burn-0.3.1/burn/chain-archive.c	2025-11-21 08:50:48.988026017 +0100
@@ -65,7 +65,7 @@
 									   boost::bind( &recording::get_graft_point, _2 ),
 									   " "
 						  )));
-		graftPoints = format("{0} /dvd.vdr={1}/counters/standard") % graftPoints % plugin::get_config_path();
+		graftPoints = format("{0} /dvd.vdr=/var/lib/vdr/burn/counters/standard") % graftPoints;
 
 		switch (get_job().get_store_mode()) {
 		case storemode_create:
--- vdr-burn-0.3.1/burn/jobs.c.orig	2025-11-21 08:51:13.231450835 +0100
+++ vdr-burn-0.3.1/burn/jobs.c	2025-11-21 08:52:28.265419493 +0100
@@ -432,7 +432,7 @@
 	std::string job::get_archive_id()
 	{
 		std::string archive_id = "";
-		std::string counter_path = format("{0}/counters/standard") % plugin::get_config_path();
+		std::string counter_path = "/var/lib/vdr/burn/counters/standard";
 		std::ifstream f(counter_path.c_str());
 		if (!f) {
 			proctools::logger::error(proctools::format("couldn't read {0}") % counter_path);
--- vdr-burn-0.3.1/burn/Makefile.orig	2025-11-21 08:53:05.976788411 +0100
+++ vdr-burn-0.3.1/burn/Makefile	2025-11-21 08:55:09.550598298 +0100
@@ -82,7 +82,7 @@
 endif
 
 ifndef BURN_DVDDEV
-BURN_DVDDEV=/dev/dvdrw
+BURN_DVDDEV=/dev/cdrom
 endif
 
 ifndef BURN_ISODIR
@@ -90,7 +90,7 @@
 endif
 
 ### compile only with ttxtsub support if core VDR is patched
-ifneq ($(strip $(wildcard $(VDRDIR)/vdrttxtsubshooks.h)),)
+ifneq ($(strip $(wildcard /usr/include/vdr/vdrttxtsubshooks.h)),)
 DEFINES += -DTTXT_SUBTITLES
 endif
 
--- vdr-burn-0.3.1/burn/README.orig	2025-11-21 08:55:44.530592239 +0100
+++ vdr-burn-0.3.1/burn/README	2025-11-21 08:57:34.216487931 +0100
@@ -66,8 +66,8 @@
 
    Optionally you can change some defaults:
    - default temporary folder   BURN_TMPDIR (default: /tmp) must be a UNIX filesystem
-   - default DVD writer device  BURN_DVDDEV (default: /dev/dvdrw)
-   - default ISO directory      BURN_ISODIR (default: /pub/export)
+   - default DVD writer device  BURN_DVDDEV (default: /dev/cdrom)
+   - default ISO directory      BURN_ISODIR (default: /video dir)
 
    e.g.:
    # make BURN_DVDDEV=/dev/cdrecorder BURN_ISODIR=/export plugins
@@ -103,7 +103,7 @@
   -d DIR,   --datadir=DIR   use DIR for temporary datafiles
                             (default: /var/spool/video)
   -D DEV    --dvd=DEV       use DEV as DVD Writer
-                            (default: /dev/dvdrw)
+                            (default: /dev/cdrom)
   -i DIR    --iso=DIR       use DIR to store ISO images
                             (default: /pub/export)
 
--- vdr-burn-0.3.1/burn/scripts/vdrburn-archive.sh.orig	2025-11-21 08:58:07.377371906 +0100
+++ vdr-burn-0.3.1/burn/scripts/vdrburn-archive.sh	2025-11-21 09:00:21.901257997 +0100
@@ -53,7 +53,7 @@
 		RECORDING_DMH="$RECPATH"/"$UPPERRECDIR"_DVD/"$RECDIR"
 		
 		mkdir -p "$RECORDING_DMH"
-		cp "$CONFIG_PATH/counters/standard" "$RECORDING_DMH"/dvd.vdr
+		cp "/var/lib/vdr/burn/counters/standard" "$RECORDING_DMH"/dvd.vdr
 		# Old style Records ("index.vdr"/"info.vdr")
 		if [ -f "$RECORDING_PATH"/index.vdr ]; then
 			cp "$RECORDING_PATH"/info.vdr "$RECORDING_DMH"/info.vdr
@@ -66,8 +66,9 @@
 	;;
 
 	archivemark)
-		((DVD_ID = `cat $CONFIG_PATH/counters/standard | sed "s/^0*//"` + 1))
-		printf "%04d\n" $DVD_ID > "$CONFIG_PATH/counters/standard"
+		((DVD_ID = `cat /var/lib/vdr/burn/counters/standard | sed "s/^0*//"` + 1))
+		printf "%04d\n" $DVD_ID > "/var/lib/vdr/burn/counters/standard"
+
 	;;
 
 	*)
--- vdr-burn-0.3.1/burn/scripts/vdrburn-dvd.sh.orig	2025-11-21 09:00:57.222131047 +0100
+++ vdr-burn-0.3.1/burn/scripts/vdrburn-dvd.sh	2025-11-21 09:05:49.974618630 +0100
@@ -93,7 +93,8 @@
 		fi
 
 		$IO_NICE $JAVA_EXE -Djava.awt.headless=true \
-				-jar $PROJECTX_JAR \
+				-cp $PROJECTX_JAR:$(build-classpath apache-commons-net jakarta-oro) \
+				net.sourceforge.dvb.projectx.common.Start \
 				-ini $CONFIG_PATH/ProjectX.ini \
 				$TTXT_OPTS \
 				$CUT -id $USED_TRACKS \
@@ -245,15 +246,15 @@
 		echo "Deleting $TEMP_PATH/INDEX_${TRACK_ON_DVD}"
 		rm -rf "$TEMP_PATH/INDEX_${TRACK_ON_DVD}"
 		echo "Creating "$RECORDING_DMH"/dvd.vdr"
-		cp "$CONFIG_PATH/counters/standard" "$RECORDING_DMH"/dvd.vdr
+		cp "/var/lib/vdr/burn/counters/standard" "$RECORDING_DMH"/dvd.vdr
 		printf "%04d\n" $(echo ${TRACK_ON_DVD} | sed 's/^0*//') >> "$RECORDING_DMH"/dvd.vdr
 	;;
 
 	archivemark)
 		echo "Creating $DVDAUTHOR_PATH/dvd.vdr"
-		cp "$CONFIG_PATH/counters/standard" "$DVDAUTHOR_PATH/dvd.vdr"
-		((DVD_ID = `cat $CONFIG_PATH/counters/standard | sed "s/^0*//"` + 1))
-		printf "%04d\n" $DVD_ID > "$CONFIG_PATH/counters/standard"
+		cp "/var/lib/vdr/burn/counters/standard" "$DVDAUTHOR_PATH/dvd.vdr"
+		((DVD_ID = `cat /var/lib/vdr/burn/counters/standard | sed "s/^0*//"` + 1))
+		printf "%04d\n" $DVD_ID > "/var/lib/vdr/burn/counters/standard"
 	;;
 
 	mkiso)
