diff -up burn-0.1.0-pre21/burn.c~ burn-0.1.0-pre21/burn.c
--- burn-0.1.0-pre21/burn.c~	2006-09-16 21:33:36.000000000 +0300
+++ burn-0.1.0-pre21/burn.c	2008-04-22 21:38:55.000000000 +0300
@@ -45,8 +45,8 @@ namespace vdr_burn
 
 	string plugin::get_character_encoding()
 	{
-#ifdef UTF8PATCH
-		return "utf8";
+#if VDRVERSNUM >= 10510 || defined(UTF8PATCH)
+		return "utf-8";
 #else
 		return tr("iso8859-15");
 #endif
